README.md

```diff
@@ -134,6 +153,8 @@ export default ExamplePage;
 
 The same approach will work for [ISR (Incremental Static Regeneration)](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration).
 
+Both `getDefinitions()` and `getFrontendFlags()` can take arguments overriding URL, token and other request parameters.
```

Comment:
I'm not sure I understand this. How? Are the methods documented anywhere? Do we have examples? Ideally, I'd like a full API reference (where the API is everything exposed by this SDK [something akin to this semi-random [rust crate doc](https://docs.rs/clap/latest/clap/struct.Arg.html#implementations)]) but that may be a bit heavy for now (and not something we want to write by hand), so maybe we can find a middle ground?

Reply:
![image](https://user-images.githubusercontent.com/2625371/217311152-854e5f48-5276-4c45-894c-b1d1f051db0e.png)

We can choose something and add it to the README later. I think TS is fine at this stage.

Reply:
Sure, I can jive with that for now. Is there an auto-doc generation tool for typescript? Do you know?